executor/runtime/docker/docker_ssh.go

```diff
@@ -157,3 +168,28 @@ func addContainerSSHDConfigWithData(c runtimeTypes.Container, tw *tar.Writer, cf
 	}
 	return nil
 }
+
+func getContainerSSHPolicyHash(c runtimeTypes.Container) (string, error) {
+	metatronCreds := c.MetatronCreds()
+	if metatronCreds == nil {
+		return "", nil
+	}
+	appMetadata := metatronCreds.AppMetadata
+	if appMetadata == nil && *appMetadata == "" {
+		return "", nil
+	}
+	params, err := url.ParseQuery(*appMetadata)
```

Comment:
Whoa, the appmetadata is a url query string?

Reply:
Yup. Probably would have suggested json if we were doing this again from scratch, but it's a simple enough way to encode a bunch of key/value pairs.